src/pages/partners/affiliate-ib/_minimum-volume.tsx

```diff
@@ -67,6 +63,19 @@ const StyledHeader = styled(Header)`
         margin: 0 auto 16px;
     }
 `
+
+const SubtitleHeader = styled(Header)`
+    line-height: 36px;
```

Comment:
line-height exist when we passing `type="sub-section-title"`
Should we have different line height here?